video/video.ino

```diff
@@ -114,7 +128,16 @@ void loop() {
 }
 
 void processLoop(void * parameter) {
+#ifdef USERSPACE
+	uint32_t count = 0;
+#endif /* USERSPACE */
+
 	while (true) {
+#ifdef USERSPACE
+ 		if ((count & 0x7f) == 0) delay(1 /* -TM- ms */);

```

Comment:
bike-shed comment: I'm not keen on `if` statements that don't use a block, so I'd rather see this use a block :grin:

a slightly less bike-shedy comment is that the `0x7f` here should arguably be a constant, defined elsewhere, to allow it to potentially be changed, have the constant's name provide some meaning/context as to what this is actually for, and that the block should reset the `count` value.

this is the "No magic numbers" principle - for reference see [this stack overflow question](https://stackoverflow.com/questions/47882/what-are-magic-numbers-and-why-do-some-consider-them-bad)